### PR TITLE
Revamp the realm stats

### PR DESCRIPTION
**Pull request by jeremyfaller**
LMK what you think, happy to continue iterating.

 - Add a pie chart when a column is selected
 - Change cursor to pointer in table rows (to show the user something
   can happen when clicking).
 - Sort rows in reverse order

Updates #871

image

### CODE REVIEW

**Comment by jeremyfaller** [comment]
le ping

File: cmd/server/assets/realmadmin/show.html [hunk 1]
```diff
@@ -71,32 +74,42 @@ <h1>Realm stats</h1>
...
+    // numDays is the length of the realm chart.
+    var numDays = 0;
+
+    // drawRealmChart draws the realm line chart.
     function drawRealmChart() {
-  {{if $stats}}
-      var data = google.visualization.arrayToDataTable([
-        ['Date', 'Issued', "Claimed"],
+    {{if $stats}}
+      // Reverse the array, so the dates are in ascending order.
```
Comment (mikehelmick) [inline cmd/server/assets/realmadmin/show.html:91]: more efficient to do this server side?

Comment (mikehelmick) [reply cmd/server/assets/realmadmin/show.html:91]: or not because the data is needed in both orders on the client?

**Review by mikehelmick — APPROVED** [review]

**Comment by mikehelmick** [comment]
sorry - merge conflict now...

**Pull request merged**